fe/lib/components/QuestionnaireBuilder/QuestionnaireBuilder/FormBuilder/components/NewPrivacyConsentForm/NewPrivacyConsentForm.tsx

```diff
@@ -70,24 +71,32 @@ export default ({
             </Heading>
           </Box>
           <Controller
-            as={TextField}
+            render={(formProps) => (
```

Comment:
I like this a lot more, it looks less magic and we probably get better typing out of it.